starting-oss-projects.md

```diff
@@ -1,34 +1,155 @@
+
+The life cycle of an open source project describes the stages in which the project evolves, from its conception to its retirement or end of life stage. Typically, a project originates to solve a specific problem. It may become obsolete either because the problem does not exist anymore or because other projects are better suited to solve the problem. The figure below shows the different stages an open source project may undergo.
+
+![Typical lifecycle of an open source project](./img/OSS-lifecycle.png)
+
+### Planning or Concept Phase
+
+This is the starting point of every open source project. It can also be referred to as the “initiation phase”. Normally, at this stage, only an idea exists or a specific problem has been identified which requires solution. In this phase, the open source project typically has the following characteristics:
+
+* The problem that the project intends to solve has been clearly defined
+* There is either no source code available yet or the source code is only internally available. In the first case, the project only exists as idea; in the second case, the project may have been started as an company-internal project and has not been published yet
+* Possibly, the idea has been already shared with the community to get feedback. However, note that this sharing such ideas that so far have only been discussed company-internally requires approval in advance.
+
+Before starting a project, it is reasonable to get answers to the key questions:
+
+* Is it possible to join efforts with an existing open source project?
+* Can we launch and maintain the project using the OSS model?
+* What constitutes success? How do we measure it?
+* Can we financially sponsor the project? Do we have an internal executive champion?
+* Will the project be able to attract outside enterprise participation (from the start)?
+* Is there enough external interest to form and grow a developer community?
+
+(Source: [Linux Foundation](https://www.linuxfoundation.org/en/resources/open-source-guides/starting-an-open-source-project/))
+
+In addition, the following aspects should be considered in the planning phase:
+
+* What is the goal of the project and will it solve the problem?
+* Are there enough resources not only to start, but to support the project in the long-term? (You also need to obtain and ensure sponsorship)
+* Does it make sense to donate the code to a vendor-neutral, non-profit organization (that is, an open source foundation), or retain some control by owning and running the project under the responsibility of your company? Note that this decision depends on the project and may also be taken later in the life cycle. Typically, a project first needs to be published and generate interest in the community before it is handed over to a third-party organization.
+* Launch check list. For example, see the [checklist of the Linux Foundation](https://www.linuxfoundation.org/en/resources/open-source-guides/starting-an-open-source-project/#checklist)
```

Comment:
'Additional to the checklist I think we can also mention:
ECC 
IP
Trademark (e.g. check that no existing trademark is used and concept whether to design a logo and whether to apply for trademark protection)
topics,

Reply:
Will add that